<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpAbout</class>
 <widget class="QDialog" name="HelpAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>278</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WMeter - About</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>WStreamLab.ico</normaloff>WStreamLab.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Common styles */
* {
    font-family: Verdana, Arial, sans-serif; /* Fallback to Arial or generic sans-serif if Verdana is not available */
    font-size: 12pt; /* Base font size */
}

/* QPushButton */
QPushButton {
    padding: 5px 10px;
    border: 2px solid #ccc; /* Light gray border */
    background-color: #f0f0f0; /* Light gray background */
    border-radius: 5px; /* Rounded corners */
}

QPushButton:hover {
    background-color: #d7d9d7; /* Light gray background on hover */
    border: 2px solid #aaa; /* Darker gray border on hover */
}

/* QComboBox */
QComboBox {
    border: 2px solid #ADD8E6; /* Light blue border */
    border-radius: 4px; /* Rounded corners */
    padding: 2px;
    min-height: 20px; /* Adjust the minimum height */
    background-color: #f0f0f0; /* Light gray background */
}

QComboBox QAbstractItemView {
    border: 2px solid #ADD8E6; /* Light blue border for the dropdown */
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px; /* Adjust width as needed */
    border-left: 1px solid #ADD8E6; /* Light blue border for the drop-down button */
}

QComboBox::down-arrow {
    width: 0;
    height: 0;
}

QComboBox::down-arrow:disabled {
    image: url(circle.png); /* Replace with your own circle image */
    width: 16px; /* Adjust width of circle */
    height: 16px; /* Adjust height of circle */
}

/* QLineEdit */
QLineEdit {
    border: 2px solid #ADD8E6; /* Light blue border */
    border-radius: 4px; /* Rounded corners */
    height: 28px; /* Adjusted height */
    padding: 2px; /* Adjust padding if needed */
    background-color: #f0f0f0; /* Light gray background */
}

/* QGroupBox */
QGroupBox {
    border: 2px solid black;
    border-color: rgba(0, 0, 0, 0.36); /* Black with transparency */
    background-color: #E6E6FF; /* Light purple background */
    font-family: Verdana, Arial, sans-serif; /* Use Verdana for group box */
    font-size: 12pt; /* Font size */
    margin-top: 10px;
    border-radius: 8px; /* Rounded corners */
    padding-top: 10px; /* Padding for the top of the group box */
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: 5px; /* Adjusted padding for better appearance */
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="grAbout">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="lbName">
              <property name="minimumSize">
               <size>
                <width>195</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>195</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbNameValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="lbCopyright">
              <property name="minimumSize">
               <size>
                <width>195</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>195</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbCopyrightValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="lbEmail">
              <property name="minimumSize">
               <size>
                <width>195</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>195</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbEmailValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="lbAddress">
              <property name="minimumSize">
               <size>
                <width>195</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>195</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbAddressValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="lbAuthor">
              <property name="minimumSize">
               <size>
                <width>195</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>195</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbAuthorValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbClose">
         <property name="styleSheet">
          <string notr="true"> padding-left: 10px; 
 padding-top: 1px; 
 padding-right: 10px;
 padding-down: 10px; </string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
